### DISCUSSION TITLE
problem when switching between windows (other applications) or resizing the application window.

### DISCUSSION THREAD

**Discussion by wnunezc** [cefsharp/CefSharp#4166] [General]
hello, I have created an application... in C# and winform that implements cefsharp...

I have deployed the control in the following way; I haven't done anything else to it, nor have I implemented any handler or other tool...

```c#
        public void InitializeChromium()
        {

            CefSettings settings = new CefSettings();
            Cef.EnableHighDPISupport();
            Cef.Initialize(settings);
            chromeBrowser = new ChromiumWebBrowser("https://universe.flyff.com/play");
            Controls.Add(chromeBrowser);
            chromeBrowser.Dock = DockStyle.Fill;

        }

```

The problem is the following:
When I switch from one application window to another or when I maximize/minimize or resize the window, the control I add as a browser loses focus/interactions... 
In this case the url is a FlyFF game which depends on the keyboard for most interactions ... as you will see the problem is that when you return to the application these keyboard interactions are lost.

**Comment by amaitland** [comment]
There are working examples provided at https://github.com/cefsharp/CefSharp.MinimalExample

I'd suggest you test with and use one of those as a reference.

> I have deployed the control in the following way; I haven't done anything else to it, nor have I implemented any handler or other tool...

You need to add an `app.manifest` to make sure your app will work correctly see https://github.com/cefsharp/CefSharp/blob/master/NuGet/Readme.txt#L13

Your usage of `Cef.EnableHighDPISupport();` is also unlikely to work. See https://github.com/cefsharp/CefSharp/wiki/General-Usage#high-dpi-displayssupport

**Reply by wnunezc** [reply]
i have solvedit using a propietary code to keyhub al key event from the form to the browser control ...

**Reply by amaitland** [reply]
I'd still suggest you test with the `MinimalExample`. No such workaround should be necessary.

You can manually call https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.control.focus?view=windowsdesktop-6.0 to give the browser focus.

**Reply by wnunezc** [reply]
not work

**Reply by amaitland** [reply]
https://idownvotedbecau.se/itsnotworking/
